test/index.js

```diff
@@ -88,3 +88,10 @@ test('throw', function(done) {
   })
 });
 
+test('abort', function() {
```

Comment:
this test should use the `done` callback to wait a few moments to make sure that `test/hello.js` doesn't actually load (and your assertion is not called).